### PR TITLE
supported search params - take 1

### PR DESCRIPTION
**Pull request by isaacvetter**

### CODE REVIEW

File: input/pagecontent/fetching.md [hunk 1]
```diff
@@ -46,6 +41,35 @@ The following resources can be searched to access patient information:
 
 -- table of resources and search parameters -
 
+|Resource | Supported search params|
+| ------- | ---------------------- | 
+| AllergyIntolerance | clinical-status, patient, patient+clinical+status |
+| AuditEvent | entity, target, date |
+| CarePlan | category, date, patient, status patient+category, patient+category+status+date, patient+category+status, patient+category+date	 |
```
Comment (mrinnetmaki) [inline input/pagecontent/fetching.md:48]: I'm not sure how to read list items like `status patient+category`. There are a few items like that. Are they just misssing a comma, should it read `category, date, patient, status, patient+category, patient+category+status+date, patient+category+status, patient+category+date`?

**Review by mrinnetmaki — COMMENTED** [review]
Added a question on how to read the list.

File: input/pagecontent/fetching.md (hunk 1 above)
Comment (isaacvetter) [reply input/pagecontent/fetching.md]: It's possible that we'll remove this resource. 
 It's likely that there was intended to be a `+` on `status patient+category`

Comment (mrinnetmaki) [reply input/pagecontent/fetching.md]: There are similar markings on
- line 50, `code patient+code`
- line 51, `type patient+type`
- line 52, `date patient+code+date`
- line 53, `period patient+category`
- line 54, `type date+patient`
- line 55, `target-date patient+lifecycle-status`
- ...
In fact, on nearly every line, and in the same place.

My strong guess is that there should be a comma in each of these lines, and it has just been stripped by mistake in some copy-paste operation.

**Pull request merged**